src/server/controllers/accelerometer.py

```diff
@@ -0,0 +1,39 @@
+import datetime
+from models.models import AccelerometerAcquisition
+from schemas.accelerometer import AccelerometerSchema
+from schemas.filter import Filter
+from sqlalchemy.orm import Session
+from typing import List
+from fastapi import status
+from fastapi.responses import JSONResponse
+
+
+def create_accelerometer_record(accelerometer_schema: List[AccelerometerSchema], db: Session):
+    records = [AccelerometerAcquisition(**accelerometer_record.dict()) for accelerometer_record in accelerometer_schema]
+    db.add_all(records)
+    db.commit()
```

Comment:
Isso pode gerar alguma exceção, proteger com um try e adicionar um except para quando a exceção ocorrer, por exemplo, ocorrendo alguma exceção retornar o código abaixo:

return JSONResponse(         status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
            content={"error": "Failed to insert data to the database."})